src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -21,13 +23,23 @@ const GovernanceReviewTeam = () => {
   const dispatch = useDispatch();
   const { systemId, activePage } = useParams();
 
+  const systemIntake = useSelector(
+    (state: AppState) => state.systemIntake.systemIntake
+  );
+
+  const businessCase = useSelector(
+    (state: AppState) => state.businessCase.form
+  );
+
   useEffect(() => {
     dispatch(fetchSystemIntake(systemId));
   }, [dispatch, systemId]);
 
-  const systemIntake = useSelector(
-    (state: AppState) => state.systemIntake.systemIntake
-  );
+  useEffect(() => {
+    if (systemIntake.businessCaseId) {
+      dispatch(fetchBusinessCase(systemIntake.businessCaseId));
+    }
+  }, [dispatch, systemIntake]);
```

Comment:
Would it be better to use the business case id as the data point driving the hook? It might not matter here. The system intake as a whole is likely to not change all that often.